.vsts-dotnet-ci.yml

```diff
@@ -14,40 +14,40 @@ variables:
   value: none
 
 jobs:
-# - job: CheckVersionBumpOnReleaseBranches
```

Comment:
@JaynieBai could you please try to disable this step conditionally it we execute on main branch?

Reply:
@JaynieBai thank you!